samples/snippets/src/main/java/com/example/analytics/RunReportWithPropertyQuotaSample.java

```diff
@@ -0,0 +1,93 @@
+ */
+
+// [START analyticsdata_run_report_with_property_quota]
+
+import com.google.analytics.data.v1beta.BetaAnalyticsDataClient;
+import com.google.analytics.data.v1beta.DateRange;
+import com.google.analytics.data.v1beta.Dimension;
+import com.google.analytics.data.v1beta.Metric;
+import com.google.analytics.data.v1beta.RunReportRequest;
+import com.google.analytics.data.v1beta.RunReportResponse;
+
+public class RunReportWithPropertyQuotaSample {
+
+  public static void main(String... args) throws Exception {
+    // TODO(developer): Replace with your Google Analytics 4 property ID before running the sample.
+    String propertyId = "YOUR-GA4-PROPERTY-ID";
+    sampleRunReportWithPropertyQuota(propertyId);
+  }
+
+  // Runs a report and prints property quota information.
+  static void sampleRunReportWithPropertyQuota(String propertyId) throws Exception {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (BetaAnalyticsDataClient analyticsData = BetaAnalyticsDataClient.create()) {
+      RunReportRequest request =
+          RunReportRequest.newBuilder()
+              .setProperty("properties/" + propertyId)
+              .setReturnPropertyQuota(true)
+              .addDimensions(Dimension.newBuilder().setName("country"))
+              .addMetrics(Metric.newBuilder().setName("activeUsers"))
+              .addDateRanges(DateRange.newBuilder().setStartDate("7daysAgo").setEndDate("today"))
+              .build();
+
+      // Make the request.
+      RunReportResponse response = analyticsData.runReport(request);
+
+      // [START analyticsdata_run_report_with_property_quota_print_response]
+      if (response.hasPropertyQuota()) {
+        System.out.printf("Tokens per day quota consumed: %d, remaining: %d.%n",
+            response.getPropertyQuota().getTokensPerDay().getConsumed(),
+            response.getPropertyQuota().getTokensPerDay().getRemaining());
+        System.out.printf("Tokens per hour quota consumed: %d, remaining: %d.%n",
+            response.getPropertyQuota().getTokensPerHour().getConsumed(),
+            response.getPropertyQuota().getTokensPerHour().getRemaining());
+        System.out.printf("Concurrent requests quota consumed: %d, remaining: %d.%n",
+            response.getPropertyQuota().getConcurrentRequests().getConsumed(),
+            response.getPropertyQuota().getConcurrentRequests().getRemaining());
+        System.out.printf("Server errors per project per hour quota consumed: %d, remaining: %d.%n",
+            response.getPropertyQuota().getServerErrorsPerProjectPerHour().getConsumed(),
+            response.getPropertyQuota().getServerErrorsPerProjectPerHour().getRemaining());
```

Comment:
nit: it's preferable to line-break consistently at the higher syntactic level
```suggestion
        System.out.printf(
            "Tokens per day quota consumed: %d, remaining: %d.%n",
            response.getPropertyQuota().getTokensPerDay().getConsumed(),
            response.getPropertyQuota().getTokensPerDay().getRemaining());
        System.out.printf(
            "Tokens per hour quota consumed: %d, remaining: %d.%n",
            response.getPropertyQuota().getTokensPerHour().getConsumed(),
            response.getPropertyQuota().getTokensPerHour().getRemaining());
        System.out.printf(
            "Concurrent requests quota consumed: %d, remaining: %d.%n",
            response.getPropertyQuota().getConcurrentRequests().getConsumed(),
            response.getPropertyQuota().getConcurrentRequests().getRemaining());
        System.out.printf(
            "Server errors per project per hour quota consumed: %d, remaining: %d.%n",
            response.getPropertyQuota().getServerErrorsPerProjectPerHour().getConsumed(),
            response.getPropertyQuota().getServerErrorsPerProjectPerHour().getRemaining());
```